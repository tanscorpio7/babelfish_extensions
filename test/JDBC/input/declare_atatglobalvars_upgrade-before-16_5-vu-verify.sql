insert into t1_declare_atatglobalvars_upgrade values(4)
go
exec p1_declare_atatglobalvars_upgrade
go
select dbo.f1_declare_atatglobalvars_upgrade() 
go
