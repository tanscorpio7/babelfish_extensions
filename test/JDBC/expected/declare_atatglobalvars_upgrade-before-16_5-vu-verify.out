insert into t1_declare_atatglobalvars_upgrade values(4)
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: Incorrect syntax near '@@max_precision'.)~~

exec p1_declare_atatglobalvars_upgrade
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: Incorrect syntax near '@@max_precision'.)~~

select dbo.f1_declare_atatglobalvars_upgrade() 
go
~~START~~
int
~~ERROR (Code: 33557097)~~

~~ERROR (Message: Incorrect syntax near '@@max_precision'.)~~

